docker_cli/_exceptions.py

```diff
@@ -0,0 +1,125 @@
+from typing import Optional, overload
+
+
+class CommandNotFoundError(Exception):
+    """Raised when a command is attempted but not found."""
+
+    def __init__(self, command_name: str):
+        """
+        Raise this exception when a command is attempted but not found.
+
+        Parameters
+        ----------
+        command_name : str
+            The name of the command.
+        """
+        self.command_name = command_name
+        super(CommandNotFoundError, self).__init__(
```

Comment:
It's typically not necessary to pass arguments to `super()` in Python3

https://stackoverflow.com/a/10483204

```suggestion
        super().__init__(
```